ydb/core/tx/schemeshard/schemeshard__operation_apply_build_index.cpp

```diff
@@ -109,7 +117,7 @@ TVector<ISubOperation::TPtr> CancelBuildIndex(TOperationId nextId, const TTxTran
         Y_ABORT_UNLESS(index.Base()->GetChildren().size() == 1);
         for (auto& indexChildItems: index.Base()->GetChildren()) {
             const TString& implTableName = indexChildItems.first;
-            Y_ABORT_UNLESS(implTableName == "indexImplTable", "unexpected name %s", implTableName.c_str());
+            Y_ABORT_UNLESS(NTableIndex::IsImplTable(implTableName), "unexpected name %s", implTableName.c_str());
```

Comment:
it's incorrect (abort above), I rewrote this code in other PR

Reply:
How bad is it? May be it should be corrected here, to avoid commit with "incorrect" behaviour?

Reply:
Check assert above about count of impl tables

Reply:
It was line 109

Reply:
Fixed to ` size() >= 1);`